app/mailers/notifier.rb

```diff
@@ -10,10 +10,9 @@ class Notifier < ActionMailer::Base
 
   # Welcome user, login credentials.  CC to PI and Department Admin.
   # Who created the account.  How to update.
-  def new_user(args)
-    @user = args[:user]
-    @password = args[:password]
-    send_nucore_mail args[:user].email, text("views.notifier.new_user.subject")
+  def new_user(user:)
```

Comment:
Just doing some testing. We do need to pass the password directly in here. When the user gets serialized for delayed_job, it doesn't serialize `password`, only the salt and `encrypted_password`. So when delayed_job comes to pick it up, `User#password` doesn't exist on the object, so it's not included with the email.